rectools/model_selection/cross_validate.py

```diff
@@ -24,6 +25,26 @@
 from .splitter import Splitter
 
 
+@contextmanager
+def compute_timing(label: str, timings: tp.Optional[tp.Dict[str, float]] = None) -> tp.Iterator[None]:
+    """
+    Context manager to compute timing for a code block.
+
+    Parameters
+    ----------
+    label : str
+        Label to store the timing result in the timings dictionary.
+    timings : dict, optional
+        Dictionary to store the timing results. If None, timing is not recorded.
+    """
+    if timings is not None:
+        start_time = time.time()
+        yield
+        timings[label] = round(time.time() - start_time, 5)
```

Comment:
Please don't do this

1. If we need to format the values somehow it should always be done separately. We should separate the computing level and presentation level. From the computing level we should always return the raw values.
2. In this specific case I think we shouldn't format the value at all. I don't see much sense in it, also we're not doing this for other metrics.